### PR TITLE
Info panel UI styling and small UI fixes

### PR DESCRIPTION
**Pull request by sandrasong**
* Separate and style activity data in MyShare panel

* Apply consistent bold and $ style to all info panels

FireShot Capture 275 - Saddle - http___localhost_3000_#_deposit

* Fix tooltip arrow color (Infinite Approval)
**Before**
  Screen Shot 2021-01-19 at 6 20 42 PM

**After**
  Screen Shot 2021-01-19 at 7 02 54 PM

* Fix LP_staking_banner outline color in dark mode
**Before**
  stake_banner_before

**After**
  stake_banner_after

* Add space in GasField and SlippageField
  Screen Shot 2021-01-19 at 6 47 24 PM

**Final screenshot**
01:20_panel update

### CODE REVIEW

File: src/components/MyActivityCard.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    margin-bottom: 24px;
+    font-size: 30px;
+    font-weight: $bold;
+    color: var(--text-title);
+  }
+
+  div.activityTable {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 48% 52%;
```
Comment (hammeiam) [inline src/components/MyActivityCard.scss:23]: any reason this can't be 50/50?

File: src/components/PoolInfoCard.tsx [hunk 2]
```diff
@@ -78,8 +72,9 @@ function PoolInfoCard({ data }: Props): ReactElement {
           {formattedData.tokens.map((token, index) => (
             <div className="token" key={index}>
               <img alt="icon" src={token.icon} />
-              <span>{token.name}</span>
-              <span className="tokenPercent">{token.percent}%</span>
+              <span className="bold">
+                {token.name + " " + token.percent + "%"}
```
Comment (hammeiam) [inline src/components/PoolInfoCard.tsx:76]: prefer interpolation for this whole component `{'${token.name} ${token.percent}%'}`

File: src/components/MyActivityCard.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            <span className="label">USD {t("deposit")}</span>
+            <span>{"$" + historicalFormattedData.totalDepositsUSD}</span>
+          </div>
+          <div key="withdrawals-btc">
+            <span className="label">BTC {t("withdrawal")}</span>
+            <span>{historicalFormattedData.totalWithdrawalsBTC}</span>
+          </div>
+          <div key="withdrawals-usd">
+            <span className="label">USD {t("withdrawal")}</span>
+            <span>{"$" + historicalFormattedData.totalWithdrawalsUSD}</span>
```
Comment (hammeiam) [inline src/components/MyActivityCard.tsx:59]: prefer interpolation over casting

**Review by hammeiam — APPROVED** [review]
small nits, looks good!

**Comment by mehmeta** [comment]
If the USD profit is negative, it displays as `$-X`, I think we'd prefer it to be `-$X`. Otherwise LGTM

**Comment by sandrasong** [comment]
Screen Shot 2021-01-18 at 4 14 09 PM

@mehmeta this was what I saw yesterday. Can you fix the logic if the negative is supposed to be `$-X`?

File: src/components/MyActivityCard.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        totalWithdrawalsBTC: parseFloat(
+          formatUnits(historicalPoolData.totalWithdrawalsBTC, 36),
+        ).toFixed(5),
+        totalProfitBTC: parseFloat(
+          formatUnits(historicalPoolData.totalProfitBTC, 36),
+        ).toFixed(5),
+        totalDepositsUSD: parseFloat(
+          formatUnits(historicalPoolData.totalDepositsUSD, 36),
+        ).toFixed(),
+        totalWithdrawalsUSD: parseFloat(
```
Comment (mehmeta) [inline src/components/MyActivityCard.tsx:31]: Moving toFixed to HTML entirely for all the different values would be better for consistency here I think.

**Review by mehmeta — APPROVED** [review]

**Pull request merged**